pkg/registry/scripts/projects.js

```diff
@@ -98,7 +105,39 @@
             function createDialog() {
                 return $modal.open({
                     controller: 'ProjectNewCtrl',
-                    templateUrl: 'views/project-new-dialog.html',
+                    templateUrl: 'views/project/add-project-dialog.html',
+                });
+            }
+
+            return {
+                create: createDialog,
+            };
+        }
+    ])
+
+    .factory('userActions', [
```

Comment:
Lets just put this into projectActions. That is: the actions for the projects page. And not have separate userActions, groupActions.